### ISSUE TITLE
Major Game issues

### ISSUE THREAD

**Issue by thatpig** [dimok789/loadiine_gx2#114]
For the most part it runs well and games work, but for some reason mario kart 8 freezes on boot no matter what I do. Tried booting in smash mode. All the files are correctly organized. The only reason for this I could see is because I have the EUR region version trying to play it on a USA console, but according to the compatibility list EUR MK8 should run fine on a USA console. I can't find MK8 USA files to save my life anyway. Also my friend who is still using an older build of loadiine can run it fine. I'm stumped on what the issue is, but it seems this most recent update of loadiine has caused it.

**Comment by CreeperMario** [comment]
If Mario Kart 8 works on older versions of Loadiine GX2, then perhaps try using an older version.
From what I know, Mario Kart 8 works best without any updates or DLC, if you have the updates and DLC in the package, the game must be booted with Smash Bros.

Although I wouldn't know, I only use Loadiine for New Super Mario Bros. U modding.

**Comment by thatpig** [comment]
I see you're not the first to say something along those lines. Booting with smash bros has given me the same result, no matter how I go about it, but according to what you and others have said if I remove the updates in it, theoretically it should work then. I need to figure out how to remove updates out though and I'm not sure what or where the updates are.

**Comment by CreeperMario** [comment]
Yeah, in order to do that you would need to do one of the following:
1. Remove the updates, DLC and save data from your console, and re-dump the game using DDD.
2. Download v0 of the game from NUS and blindly decrypt it using a ticket you can't even dump yet (unless I'm missing something in this step).

I don't see why booting in Smash Bros. mode isn't working. Loadiine GX2 has a debugging server that can be access using a tool hidden in the repo somewhere, although I couldn't get it to work myself (Mac). That might give us something useful if we can get it working.

If not, another option may be to boot the game using one of the other two modes, one of which requires downloading KaraokeU (or something like that) from the eShop, and the other requires the Art Ateiler game disc (again, not too sure on the name).

**Comment by castorfou** [comment]
> Loadiine GX2 has a debugging server that can be access using a tool hidden in the repo somewhere, although I couldn't get it to work myself (Mac). That might give us something useful if we can get it working.
Do you have a tuto explaining how to use it?
I have the same problem as @thatpig , and would like to help solving it.

**Comment by CreeperMario** [comment]
@castorfou no, I could not find a tutorial on it. As I said, I was unable to actually get it working within the bash terminal on Mac.

**Comment by dibas** [comment]
The tool in question can be found here: https://github.com/dimok789/loadiine_gx2/tree/master/server
It logs file I/O events, pretty much. It might help you in solving your problem, if there's a file missing on your end for example. Don't forget to enable the server connection in loadiine's settings and set the correct ip address. 

This does not sound like a problem with Loadiine to me, but to make that sure please keep us updated on your progress.

**Comment by thatpig** [comment]
I never realized I got all these responses until now that I've returned to trying to solve this. where do I add these files to?
